src/Equinor.ProCoSys.IPO.Domain/DateTimeExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Equinor.ProCoSys.IPO.Domain
+{
+    public static class DateTimeExtensions
+    {
+        private static readonly DayOfWeek firstDayOfWeek = DayOfWeek.Monday;
+
+        public static DateTime AddWeeks(this DateTime dateTime, int weeks)
+            => dateTime.AddDays(7 * weeks);
+
+        public static DateTime StartOfWeek(this DateTime dt)
```

Comment:
Add 3 unit tests. Pick a time on a Monday and Sunday (these tests exists in pres)